src/services/review/ReviewRequestService.ts

```diff
@@ -840,6 +836,7 @@ export default class ReviewRequestService {
     )
 
     await this.apiService.mergePullRequest(repoNameInGithub, pullRequestNumber)
+    await this.apiService.mergeStagingToMaster(repoNameInGithub)
```

Comment:
question(blocking): correct me if I'm wrong here, but would this create two separate merge commits, one on GitHub and one locally, resulting in divergent master branches?

Reply:
yea actually you're right, changed it to pull from master instead, can remove pull request from the whole flow in a separate PR

https://github.com/isomerpages/isomercms-backend/pull/1172/commits/33e665e414587f170036520bd69ac4c0bb619087